lib/reek/smells/utility_function.rb

```diff
@@ -68,7 +68,7 @@ def inspect(ctx)
           context: ctx,
           lines: [ctx.exp.line],
           message: "doesn't depend on instance state (maybe move it to another class?)",
-          parameters: { name: ctx.full_name })]
+          parameters: { name: ctx.name.to_s })]
```

Comment:
I'd rather go the other way round: Before this change it was a bug.
Because before `name` would have been "Foo::Bar#my_method" for instance.
Now it's just "my_method" and the client can decide if he wants to use the `context` or the `name`.